### ISSUE TITLE
Sir could you please guide of how to use this extension on Microsoft Edge

### ISSUE THREAD

**Issue by psvita123** [tushar9989/netflix-bitrate#1]
Dear Sir,
I would like to thank you for your effort.
I wanted this feature back on Microsoft Edge or The Netflix Windows App because only these two browsers support 4k streaming. Your help would be highly appreciated.
Regards.

**Comment by tushar9989** [comment]
Have you tried porting the extension using the tool Microsoft provides? 
https://docs.microsoft.com/en-us/microsoft-edge/extensions/guides/porting-chrome-extensions

**Comment by psvita123** [comment]
yes sir did that, the problem which occurred is after loading the new extension for edge and opening any video on netflix it will never play.
 to help  you understand let me say it again
The extension is loaded successfully
i go to netflix
start a video
(if it was successful the video would start buffering)
but now a blank black just shows up which stays like that for like a minute or two and then this error pops on the screen
Error Code: UI3012

**Comment by psvita123** [comment]
if possible could you try it yourself as I have no knowledge of such stuff and only by watching a video i was able to make this edge extension, which even i am not sure was correctly compiled or not.

**Comment by tushar9989** [comment]
Did not work for me either. Will look into it when I have more time and get back.

**Comment by psvita123** [comment]
alright sir i will be waiting for your reply anxiously.

**Comment by Jowat97** [comment]
@tushar9989 There is no need to rewrite this Extension for Edge, because the new Edge Version is running with Chromium Engine: https://www.microsoftedgeinsider.com/en-us/download
It is possible to run this Extension on Edge (Chromium). However, unfortunately it bricks Netflix. You can't watch any Videos on Netflix, if you use Edge (Chromium) and have this Extension enabled. You will always get an Error.

Could you maybe fix this error? It would be so amazing to have this Extension to work for Edge (Chromium), because it's the only Browser which supports 4K and HDR.

**Comment by tushar9989** [comment]
I don't have a Netflix subscription anymore and will not be able to fix the issue. You can probably fork it / send me a PR.

**Comment by Jowat97** [comment]
@tushar9989 Unfortunately my JS Knowledge isn't the best, so I think I wouldn't be able to fix this myself. However, even without a Netflix Subscriptions you can watch Trailers on Netflix. Therefor Netflix uses the same Webplayer as for regular Series. For example you could try this: https://www.netflix.com/title/80021955
Many people would really appreciate your help on this work. 👍

**Comment by iNViTiON** [comment]
FYI. The mini preview from main page and the trailer can work fine with Ctrl+Alt+Shift+S. Just the watching page that crash the player.
